parlai/crowdsourcing/utils/mturk.py

```diff
@@ -10,7 +10,9 @@
 
 from mephisto.core.hydra_config import RunScriptConfig
 from mephisto.data_model.database import MephistoDB
-from mephisto.providers.mturk.utils.script_utils import direct_soft_block_mturk_workers
+from mephisto.abstractions.providers.mturk.utils.script_utils import (
```

Comment:
Hmm - since this updated import means that we'll need to use a new version of Mephisto, it'd be good to bump the Mephisto version number in `.circleci/config.yml` now to ensure that tests pass

Reply:
Updating for now, but will likely need to use v0.2.2 once the other PRs mentioned in here are landed.